Library/Formula/devd.rb

```diff
@@ -0,0 +1,132 @@
+  homepage "https://github.com/cortesi/devd"
+  url "https://github.com/cortesi/devd/archive/v0.3.tar.gz"
+  sha256 "e806421a4de6572eb3196b215a86a3b72bcd0f5a12956224d191e47663f9c4ab"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/GeertJohan/go.rice" do
+    url "https://github.com/GeertJohan/go.rice.git", :revision => "ada95a01c963696fb73320ee662195af68be81ae"
+  end
+
+  # go.rice dependencies
+  go_resource "github.com/daaku/go.zipexe" do
+    url "https://github.com/daaku/go.zipexe.git", :revision => "a5fe2436ffcb3236e175e5149162b41cd28bd27d"
+  end
+
+  go_resource "github.com/kardianos/osext" do
+    url "https://github.com/kardianos/osext.git", :revision => "6e7f843663477789fac7c02def0d0909e969b4e5"
+  end
+
+  go_resource "github.com/GeertJohan/go.incremental" do
+    url "https://github.com/GeertJohan/go.incremental.git", :revision => "92fd0ce4a694213e8b3dfd2d39b16e51d26d0fbf"
+  end
+
+  go_resource "github.com/akavel/rsrc" do
+    url "https://github.com/akavel/rsrc.git", :revision => "ba14da1f827188454a4591717fff29999010887f"
+  end
+
+  go_resource "github.com/jessevdk/go-flags" do
+    url "https://github.com/jessevdk/go-flags.git", :revision => "fc93116606d0a71d7e9de0ad5734fdb4b8eae834"
+  end
+
+  # devd dependencies
+  go_resource "github.com/bmatcuk/doublestar" do
+    # v1.0.1
+    url "https://github.com/bmatcuk/doublestar.git", :revision => "4f612bd6c10e2ef68e2ea50aabc50c3681bbac86"
+  end
+
+  go_resource "github.com/dustin/go-humanize" do
+    url "https://github.com/dustin/go-humanize.git", :revision => "64dbdae0d393b7d71480a6dace78456396b55286"
+  end
+
+  go_resource "github.com/fatih/color" do
+    url "https://github.com/fatih/color.git", :revision => "9aae6aaa22315390f03959adca2c4d395b02fcef"
+  end
+
+  go_resource "github.com/goji/httpauth" do
+    url "https://github.com/goji/httpauth.git", :revision => "c1b2bcd8769bd15cc56751223fd4b9f45ca987ca"
+  end
+
+  go_resource "github.com/gorilla/websocket" do
+    url "https://github.com/gorilla/websocket.git", :revision => "361d4c0ffd78338ebe0a9e6320cdbe115d7dc026"
+  end
+
+  go_resource "github.com/juju/ratelimit" do
+    url "https://github.com/juju/ratelimit.git", :revision => "772f5c38e468398c4511514f4f6aa9a4185bc0a0"
+  end
+
+  go_resource "github.com/mitchellh/go-homedir" do
+    url "https://github.com/mitchellh/go-homedir.git", :revision => "d682a8f0cf139663a984ff12528da460ca963de9"
+  end
+
+  go_resource "github.com/rjeczalik/notify" do
+    url "https://github.com/rjeczalik/notify.git", :revision => "1869adb163fffce8fb5b8755379d1042cdb4c4f8"
+  end
+
+  go_resource "github.com/toqueteos/webbrowser" do
+    # v1.0
+    url "https://github.com/toqueteos/webbrowser.git", :revision => "21fc9f95c83442fd164094666f7cb4f9fdd56cd6"
+  end
+
+  go_resource "github.com/alecthomas/template" do
+    url "https://github.com/alecthomas/template.git", :revision => "b867cc6ab45cece8143cfcc6fc9c77cf3f2c23c0"
+  end
+  go_resource "github.com/alecthomas/units" do
```

Comment:
Please leave an empty line between the resources. Helps with readability, a bit.